docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: "3.8"
+services:
+  stargate:
+    image: storjlabs/stargate:dev
+    deploy:
+      replicas: 1
+      resources:
+        limits:
+          cpus: '1.5'
+          memory: 1024M
+        reservations:
+          cpus: '0.3'
+          memory: 512M
+      restart_policy:
+        condition: on-failure
+        delay: 5s
+        max_attempts: 3
+        window: 120s
+    # volumes:
+    #   - ./public.crt:/root/.local/share/storj/gateway/minio/certs/public.crt
+    #   - ./private.key:/root/.local/share/storj/gateway/minio/certs/private.key
```

Comment:
Should we document these bits as well in the readme?

Reply:
I'm of two minds about this, I sorta just want to put `traefik` or something in there to handle the SSL instead of doing it with the containers themselves, since it's all gonna be invalid certs in local dev anyway. but I won't have time to really do that until I'm done the loadtesting stuff, and people are asking me multiple times a day for this config to be in the repo. so I figured just leaving it as examples in the config for now and replacing it with a proper app router later is maybe ok? I'm def open to whatever you prefer.

Reply:
Totally fine by me!